modules/oadp-scheduling-backups.adoc

```diff
@@ -8,6 +8,13 @@
 
 You schedule backups by creating a `Schedule` custom resource (CR) instead of a `Backup` CR.
 
+[WARNING]
+====
+Leave enough time in your backup schedule for a backup to finish before another backup is created.
+
+For example, if a backup of a namespace typically takes 10 minutes, do not schedule backup more frequently than every 15 minutes.
+====
```

Comment:
@RichardHoch If you treat 'backup' as 'a single action of backing up', then I would say 'do not schedule backups' to be consistent.